packages/fxa-auth-server/test/local/senders/index.js

```diff
@@ -243,12 +243,12 @@ describe('lib/senders/index', () => {
 
             const args = email._ungatedMailer.postVerifyEmail.getCall(0).args;
             assert.equal(args[0].email, EMAIL, 'email correctly set');
-            assert.equal(args[0].ccEmails, undefined, 'no cc emails set');
+            assert.lengthOf(args[0].ccEmails, 1);
```

Comment:
This is an artifact of using that common reducer function, but it's a moot point. Newly verified accounts will only have one email address so `ccEmails` will never be set in practice.

Reply:
You can probably pass in a more correct `EMAILS` value here to test `ccEmails` is 0 or undefined.